python-sdk/src/astro/files/types/base.py

```diff
@@ -5,19 +5,25 @@
 
 import pandas as pd
 
+from astro.options import LoadOptions
+
 
 class FileType(ABC):
     """Abstract File type class, meant to be the interface to all client code for all supported file types"""
 
-    def __init__(self, path: str, normalize_config: dict | None = None):
+    def __init__(
+        self, path: str, normalize_config: dict | None = None, load_options: LoadOptions | None = None
```

Comment:
I think it would be clearer if the interface here explicitly had a list of `LoadOptions`

Reply:
@tatiana Wouldn't we need to update this every time we add a new file type? Would be better if we add base `FileLoadOptions` inheriting from `LoadOptions` and use it here?

Reply:
As per our discussion load_options should be a object because we never use multiple load_option for a integration.